products/BellHybrid/services/audio/VolumeFade.cpp

```diff
@@ -10,24 +10,39 @@ namespace audio
     {
         using namespace std::chrono_literals;
         constexpr auto timerName{"volumeFadeTimer"};
-        constexpr auto fadeInterval{33ms};
+        constexpr auto minFadeInterval{33ms};
         constexpr auto fadeStep{0.1f};
 
-        std::chrono::milliseconds calculateTimerPeriod(std::chrono::seconds elapsed,
-                                                       std::chrono::seconds duration,
-                                                       float currentVolume)
+        std::chrono::milliseconds calculateFadeInterval(std::chrono::seconds maxDuration,
```

Comment:
What about moving both of these functions inside `VolumeFade` class and making them private? As far as I see the arguments they take are mostly the fields of the class (apart from `timeElapsed`, which is calculated based on current time). That will greatly reduce number of arguments passed to the functions (in case of `calculateFadeInterval` to zero)